libs/packages/sam-formly/src/lib/formly-filters/advanced-filters/advanced-filters.component.ts

```diff
@@ -42,7 +47,15 @@ export class AdvancedFiltersComponent {
     const modalFields: FormlyFieldConfig[] = this.advancedFiltersService.convertToCheckboxes(
       this.fields
     );
-
+    if (this.sortMoreFilter) {
+      modalFields.sort((a, b) =>
```

Comment:
I think this would be safer / cleaner:

modalFields.sort((a, b) => a.templateOptions.label.localeCompare(b.templateOptions.label));


but I guess we could still see undefined if templateOptions doesn't exist (in either scenario)